### PR TITLE
sbt-native-packager in sbt builds

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Review by octonato — APPROVED** [review]
LGMT

I was looking in that as well. 

There is one left over, the `build.sbt` is still setting the version to `1.0`.

File: docs-source/docs/modules/how-to/examples/shopping-cart-service-scala/build.sbt [hunk 1]
```diff
@@ -28,6 +28,11 @@ val AkkaProjectionVersion = "1.0.0"
 
 enablePlugins(AkkaGrpcPlugin)
 
+dockerBaseImage := "adoptopenjdk:11-jre-hotspot"
```
Comment (ignasi35) [inline docs-source/docs/modules/how-to/examples/shopping-cart-service-scala/build.sbt:31]:
```suggestion
enablePlugins(JavaAppPackaging, DockerPlugin) 
dockerBaseImage := "adoptopenjdk:11-jre-hotspot"
```

**Review by ignasi35 — CHANGES_REQUESTED** [review]
Missing `addPlugins`. All the rest LGTM

**Review by ignasi35 — APPROVED** [review]

File: docs-source/docs/modules/how-to/examples/shopping-cart-service-scala/build.sbt (hunk 1 above)
Comment (patriknw) [reply docs-source/docs/modules/how-to/examples/shopping-cart-service-scala/build.sbt:31]: thanks

**Comment by patriknw** [comment]
> build.sbt is still setting the version to 1.0

thanks, fixed

**Pull request merged**